src/fhirtypes/dataTypes.ts

```diff
@@ -36,9 +36,9 @@ export type Coding = {
  * @see {@link http://hl7.org/fhir/R4/datatypes.html#ContactPoint}
  */
 export type ContactPoint = {
-  system?: string;
+  system?: 'phone' | 'fax' | 'email' | 'pager' | 'url' | 'sms' | 'other';
   value?: string;
-  use?: string;
+  use?: 'home' | 'work' | 'temp' | 'old' | 'mobile';
```

Comment:
Why the change here?

Reply:
More "typey"?  In the FHIR definitions, these are _required_ bindings with those values.  Since it's a fixed set of values, I thought I'd more closely reflect that in the TypeScript code.  Is it absolutely necessary?  No -- but it does help us be more "typey" and potentially avoid typos if we ever use that data in code.

If you were to reword this question as "Why the change here, but not there?" (pointing to a similar bit of code where I did not do this), then the answer is just that I did it as I encountered it.  So I ran into this code (actually while working on the import part) and updated it.